java/java.source.base/src/org/netbeans/modules/java/source/save/Reformatter.java

```diff
@@ -2682,6 +2779,11 @@ public Boolean visitCase(CaseTree node, Void p) {
         @Override
         public Boolean visitBreak(BreakTree node, Void p) {
             accept(BREAK);
+            ExpressionTree exprTree = TreeShims.getValue(node) ;
+            if (exprTree != null) {
+                space();
+                scan(exprTree, p);
```

Comment:
Is a return missing here? I believe the BreakTree.getValue() and getLabel() may in some cases return non-null value, like "break a;". It is probably OK to handle that as an expression, but presumably should not be handled as an expression and then as a label.

Reply:
Had corrected the visitBreak code in recent commit